Pylint: Fixe `no-else-*` issues

## 💡 Description
Update the codebase to not use else after return, raise, continue or break statements. `pylint` report is attached (`pylint . --disable=all --enable=no-else-continue,no-else-return,no-else-break,no-else-raise):
```
************* Module pds.naif_pds4_bundler.classes.list
pds/naif_pds4_bundler/classes/list.py:682:12: R1720: Unnecessary "else" after "raise", remove the "else" and de-indent the code inside it (no-else-raise)
pds/naif_pds4_bundler/classes/list.py:818:12: R1724: Unnecessary "elif" after "continue", remove the leading "el" from "elif" (no-else-continue)
************* Module pds.naif_pds4_bundler.classes.collection.collection
pds/naif_pds4_bundler/classes/collection/collection.py:71:12: R1723: Unnecessary "else" after "break", remove the "else" and de-indent the code inside it (no-else-break)
************* Module pds.naif_pds4_bundler.classes.product.product_metakernel
pds/naif_pds4_bundler/classes/product/product_metakernel.py:1031:12: R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)
************* Module pds.naif_pds4_bundler.utils.time
pds/naif_pds4_bundler/utils/time.py:204:4: R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)
```

--- a/src/pds/naif_pds4_bundler/classes/collection/collection.py
+++ b/src/pds/naif_pds4_bundler/classes/collection/collection.py
@@ -115,10 +115,9 @@ class Collection:
             # If the product is not in the kernel list (such as the orbnum
             # file), then use the mission observers and targets.
             #
-            else:
-                missions = [self.setup.mission_name]
-                observers = [self.setup.observer]
-                targets = [self.setup.target]
+            missions = [self.setup.mission_name]
+            observers = [self.setup.observer]
+            targets = [self.setup.target]
 
         return missions, observers, targets
 

--- a/src/pds/naif_pds4_bundler/classes/list.py
+++ b/src/pds/naif_pds4_bundler/classes/list.py
@@ -696,9 +696,9 @@ class KernelList:
                 logging.error("")
 
                 raise Exception(error)
-            else:
-                logging.info(f"     PASS with total of {num_file} entries.")
-                logging.info("")
+
+            logging.info(f"     PASS with total of {num_file} entries.")
+            logging.info("")
 
             #
             # Check list for duplicate entries

--- a/src/pds/naif_pds4_bundler/classes/list.py
+++ b/src/pds/naif_pds4_bundler/classes/list.py
@@ -820,12 +820,14 @@ class KernelList:
                     "Product not present in any kernel directory(ies)"
                 )
                 continue
-            elif not origin_paths and ".tm" in product.lower():
+
+            if not origin_paths and ".tm" in product.lower():
                 product_warnings[product].append(
                     "Meta-kernel will be generated during this run."
                 )
                 continue
-            elif len(origin_paths) > 1:
+
+            if len(origin_paths) > 1:
                 product_warnings[product].append(
                     "Product present in multiple directories:"
                 )

--- a/src/pds/naif_pds4_bundler/classes/product/product_metakernel.py
+++ b/src/pds/naif_pds4_bundler/classes/product/product_metakernel.py
@@ -1067,24 +1067,24 @@ class MetaKernelProduct(Product):
                 self.stop_time = stop_time
 
                 return
+
+            if hasattr(self.setup, "increment_start"):
+                start_time = self.setup.increment_start
             else:
-                if hasattr(self.setup, "increment_start"):
-                    start_time = self.setup.increment_start
-                else:
-                    start_time = self.setup.mission_start
+                start_time = self.setup.mission_start
 
-                if hasattr(self.setup, "increment_finish"):
-                    stop_time = self.setup.increment_finish
-                else:
-                    stop_time = self.setup.mission_finish
+            if hasattr(self.setup, "increment_finish"):
+                stop_time = self.setup.increment_finish
+            else:
+                stop_time = self.setup.mission_finish
 
-                logging.warning(
-                    f"-- No kernel(s) found to determine MK coverage. "
-                    f"Times from configuration will be used: {start_time} - {stop_time}"
-                )
+            logging.warning(
+                f"-- No kernel(s) found to determine MK coverage. "
+                f"Times from configuration will be used: {start_time} - {stop_time}"
+            )
 
-                self.start_time = start_time
-                self.stop_time = stop_time
+            self.start_time = start_time
+            self.stop_time = stop_time
 
         else:
 

--- a/src/pds/naif_pds4_bundler/utils/time.py
+++ b/src/pds/naif_pds4_bundler/utils/time.py
@@ -202,15 +202,10 @@ def ck_coverage(path, timsys="TDB", date_format="infomod2", system="UTC"):
     stop_time = max(end_points_list)
 
     if timsys == "SCLK":
-        return (start_time, stop_time)
-    else:
-        return et_to_date(
-            start_time,
-            stop_time,
-            date_format=date_format,
-            kernel_type="ck",
-            system=system,
-        )
+        return start_time, stop_time
+
+    return et_to_date(start_time, stop_time, date_format=date_format,
+                      kernel_type="ck", system=system)
 
 
 def pck_coverage(path, date_format="infomod2", system="UTC"):

